js/reference.js

```diff
@@ -794,6 +878,64 @@ var ERMrest = (function(module) {
                 return newReference;
             }
             return null;
+        },
+
+        /**
+         * HTML representation of the whole page which uses table-display annotation.
+         * For more info you can refer {ERM.reference.display}
+         * 
+         * Usage:
+         *```
+         * var content = page.content;
+         * if (content) {
+         *    console.log(content);
+         * }
+         *```
+         * @type {string|null}
+         */
+        get content() {
+            if (this._content !== null) {
+                // If display type is markdown which means row_markdown_pattern is set in table-display 
+                if (this._ref.display.type === 'markdown') {
+
+                    // If the number of records are zero then simply return the value
+                    if (!this._data || !this._data.length) {
+                        return "";
+                    }
+
```

Comment:
Maybe we should return `null` in this case? if there is nothing to display, we need to communicate to the app so this entire related entity can be ignored. @jrchudy I saw (# of records) on the record2 page, so there should be a way to determine this already, correct?